images/linux/Ubuntu1604-README.md

```diff
@@ -169,6 +169,7 @@ compiled using version 3.2.0.0 of the Cabal library )
 - kubectl (Client Version: v1.18.3)
 - helm (v3.2.3+g8f83204)
 - minikube version: v1.11.0
+- oc (Openshift CLI) version v4.6
```

Comment:
don't need to add it to Readme manually – it's done via DocumentInstalledItem function

Reply:
removed :)